### DISCUSSION TITLE
Minor improvments for theme selector

### DISCUSSION THREAD

**Discussion by YJDoc2** [NvChad/NvChad#2623] [General]
Hello! I have recently started using this, and this seems like a great project, with some really nice defaults.

There are two things I found out about theming, which I think can use some improvements :

1. Selected theme is not retained when I open theme selector by `<leader> +th` . Whenever I open the selector, theme automatically gets changed to the first theme in list, which is super annoying especially if I accidentally opened it or (as a newcomer) was just checking it out, and didn't want to change the theme.
2. It'd be great if the themes are split into dark and light modes. Even if it doesn't have two distinct sections, it'd be great if all light themes are together and dark themes are together. When first I was exploring things, and I pressed down button to go down in the list to one theme I wanted to see, there was some bad flashing due to switches between dark and light themes.

My setup is pretty much default, expect some additions for lsp and treesitter for languages I use.

If this is not something this project can control, can you point me to where this issue would be appropriate, so I can open it there? I'm quite new to neovim plugin ecoosystem, so not sure where this would fit.

Thanks again for making such cool project and maintaining it!

**Comment by siduck** [comment]
> It'd be great if the themes are split into dark and light modes. 

this is not possible i think, you will have to write 2 telescope extensions for this, one listing dark, other listing light
